app/views/catalog/record/_marc_contents_summary.html.erb

```diff
@@ -61,7 +61,12 @@
 <% if document.marc_links.supplemental.present? %>
   <dt title="Supplemental links">Supplemental links</dt>
   <dd>
-    <%= document.marc_links.supplemental.map(&:html).join("<br/>").html_safe %>
+    <% document.marc_links.supplemental.map do |link| %>
+      <%= link.html.html_safe %>
+      <% if link.stanford_only? %>
+        <span class='stanford-only'></span>
+      <% end %>
+    <% end.join("<br/>") %>
```

Comment:
```suggestion
    <% end.join("<br/>".html_safe) %>
```

I'm having trouble finding a record with multiple supplemental links, but do you think this `<br>` needs to be html safe too?

Reply:
Pushed up an alternate solution.